### ISSUE TITLE
Support for alternate textures

### ISSUE THREAD

**Issue by Echoweaver** [crazysnailboy/VillagerTrades#22]
Is it possible to create a resource pack with alternate textures for custom villagers. I attempted to set up the traditional mcpatcher directories in my resource pack to add alternate textures for tinkerer, brewer, and redstoner, but they're not respected.

If it's not supported, would you consider adding support? I use the Lithos resource pack, and I'm completely addicted to the sense of life it gives villages.

**Comment by Flytre** [comment]
Yes, I did it successfully. I don't have access to the resource pack right now, but I believe you need to change the texture at /assets/vtt/textures/entity/villager/villager_name.json.
